biorbd_optim/objective_functions.py

```diff
@@ -132,6 +133,7 @@ class Mayer(Enum):
         MINIMIZE_PREDICTED_COM_HEIGHT = (PenaltyType.MINIMIZE_PREDICTED_COM_HEIGHT,)
         ALIGN_SEGMENT_WITH_CUSTOM_RT = (PenaltyType.ALIGN_SEGMENT_WITH_CUSTOM_RT,)
         ALIGN_MARKER_WITH_SEGMENT_AXIS = (PenaltyType.ALIGN_MARKER_WITH_SEGMENT_AXIS,)
+        TRACK_CONTACT_FORCES = (PenaltyType.TRACK_CONTACT_FORCES,)
```

Comment:
Please move this right after TRACK_ALL_CONTROLS

Reply:
done